app/assets/javascripts/admin/analytics-modules/ga4-select-tracker.js

```diff
@@ -16,7 +16,8 @@ window.GOVUK.analyticsGa4.analyticsModules =
 
           if (
             event.target.tagName.toLowerCase() === 'select' &&
-            !event.target.getAttribute('hidden')
+            !event.target.getAttribute('hidden') &&
+            !event.target.closest("[data-module~='ga4-finder-tracker']")
```

Comment:
All for avoiding double-tracking, but do you have a plan for consolidating _all_ tracking onto the ga4-finder-tracker approach? It feels like a rabbit hole if we're sometimes having to disable tracking like this and sometimes not. I'd have thought / hoped that pulling in ga4-finder-tracker would give us a tracking module / wrapper that sufficiently encompasses all basic components (i.e. 'select'), if not the more complex ones ('select with search')

Reply:
My plan is to remove the select tracking entirely when I track finders and [form changes](https://github.com/alphagov/whitehall/pull/10201/files).